lib/authc/AssertionAuthenticationResult.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+var nJwt = require('njwt');
+var Account = require('../resource/Account');
+
+function AssertionAuthenticationResult(dataStore, data) {
+  this.dataStore = dataStore;
+
+  // Copy data properties.
+  if (data) {
+    for (var key in data) {
+      if (this[key]) {
```

Comment:
This key will still be copied if the value of `this[key]` is [falsey](http://stackoverflow.com/a/19839953/1685939), e.g. `0` or `false`. Is this intended or should it only apply for `undefined`/`null`/non-existent values?